synapseclient/core/download/download_functions.py

```diff
@@ -161,6 +170,105 @@ async def download_file_entity(
     entity.cacheDir = os.path.dirname(download_path)
 
 
+async def download_file_entity_model(
+    *,
+    download_location: str,
```

Comment:
There is a conditional further down that is `if download_location is not None:` - should we have this arg default to `None` to allow for this to happen?

Reply:
I'll update the type to `Union[str, None]`